arbitrator/stylus/src/lib.rs

```diff
@@ -354,3 +354,30 @@ pub unsafe extern "C" fn stylus_drop_vec(vec: RustBytes) {
         mem::drop(vec.into_vec())
     }
 }
+
+/// Gets lru cache metrics.
+#[no_mangle]
+pub extern "C" fn stylus_get_lru_cache_metrics() -> LruCacheMetrics {
+    InitCache::get_lru_metrics()
+}
+
+/// Clears lru cache.
+/// Only used for testing purposes.
+#[no_mangle]
+pub extern "C" fn stylus_clear_lru_cache() {
```

Comment:
wont stylus_set_cache_lru_capacity(0) and then resize-again work?

Reply:
Nope :(, capacity must be non zero https://docs.rs/clru/0.6.2/clru/struct.CLruCache.html#method.resize